### PR TITLE
Implement creating a new Jira issue

### PR DESCRIPTION
**Pull request by christianbumann**

### CODE REVIEW

File: src/Cake.Jira/Cake.Jira.csproj [hunk 1]
```diff
@@ -26,8 +26,6 @@
   <ItemGroup>
     <PackageReference Include="Atlassian.SDK" Version="10.4.0" />
     <PackageReference Include="Cake.Core" Version="0.28.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="RestSharp" Version="106.2.2" />
```
Comment (christianbumann) [inline src/Cake.Jira/Cake.Jira.csproj]: Both packages are coming from the [Atlassian.SDK](https://www.nuget.org/packages/Atlassian.SDK) dependencies. I got an inconsistent version error on Newtonsoft.Json if I did a test against inside the test project.

**Comment by christianbumann** [comment]
It would be nice to have a release which I can use after successful merge.

**Comment by christianbumann** [comment]
Our company is willing to contribute to this repository. It would be nice if our company could do additional functionalities without asking you every time.

**Comment by Ninglin** [comment]
The repository follows the open source methodology. You can contribute to it. However, all of the changes will have to go through the repository maintainers. Which at the moment is just me. Why close the PR?

**Pull request closed without merging**